test-five/infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,196 @@
+import * as cdk from 'aws-cdk-lib';
+import {
+  aws_certificatemanager as acm,
+  aws_cloudfront as cloudfront,
+  aws_s3 as s3,
+  aws_s3_deployment as s3Deploy,
+  aws_route53 as route53,
+  aws_route53_targets as targets,
+  aws_cloudwatch as cloudwatch,
+  aws_cloudwatch_actions as cw_actions,
+  aws_sns as sns,
+  aws_sns_subscriptions as subscriptions,
+  CfnOutput,
+  CfnParameter,
+  RemovalPolicy
+} from 'aws-cdk-lib';
+
+// Setting the domain name.
+const DOMAIN_NAME = process.env.DOMAIN_NAME!;
+
+/**
+ * Represents the CloudFormation stack for the TestFive application.
+ * @class
+ * @extends cdk.Stack
+ */
+export class TestFiveStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    /**
+     * 👉 Environment Variable:
+     */
+
+    const envParameter = new CfnParameter(this, "Environment", {
```

Comment:
+1 for environments. This hasn't really been applied right through CI/CD though right? Not saying you should do the work for this, just clarifying.

Reply:
Yes, it has already been integrated into the CI/CD workflow. Please review [this line of code](https://github.com/pablomagro/show-us-what-you-got/blob/892c3f395fc63cb15c59cc1e4e640bfc04320e5d/.github/workflows/deploy.yml#L126C31-L126C31).